tests/feature_creation_tests.py

```diff
@@ -16,4 +22,26 @@ def test_example(self):
         # Call feature creation function on the dataframes
 
         # Assert that the values are equal
-        assert True
\ No newline at end of file
+        assert True
+    def test_create_taped_stats_feats(self):
+        """
+        Tests the create_taped_stats_feats function of the TapedStats class.
+        """
+        # Call the feature creation function
+        result_df = self.taped_stats.create_taped_stats_feats(self.fights_df, self.fighter_df)
+        # Remove specified columns from expected DataFrame
+        columns_to_compare = self.expected_df.columns.difference(['fighter-a_dob', 'fighter-b_dob', 'date'])
+
+        # Ensure the same columns are being compared in both DataFrames
+        result_df = result_df[columns_to_compare]
+        expected_df = self.expected_df[columns_to_compare]
+        
+        # Compare the generated results with the expected values
+        pd.testing.assert_frame_equal(result_df, expected_df, check_dtype=False)
+
+        print("test_create_taped_stats_feats passed")
```

Comment:
Maybe make it a bit more general in case we change the name? Like ("Taped stats feature tests passed")